documentation/announcing-az-module.md

```diff
@@ -0,0 +1,163 @@
+# Announcing New Module 'Az'
+In August 2018 we released a new module, 'Az' which combines the functionality of the AzureRM and AzureRM.Netcore modules.  Az runs on both PowerShell 5.1 and PowerShell Core.  'Az' ensures that the PowerShell and PowerShell core cmdlets for managing Azure resources will always be in sync and up to date.  In addition, Az will simplify and regularize the naming of Azure cmdlets, and the organization of Azure modules.  Az is intended as a replacement for the AzureRM.Netcore and AzureRM modules.
+
+Az currently ships in CloudShell, and can be found on the PowerShell Gallery [here](https://www.powershellgallery.com/packages/Az/0.2.2)
```

Comment:
@markcowl nit: I would remove the trailing `0.2.2` from the URL

Reply:
If you just go to Az/  is takes you to version 0.1.0, inexplicably

Reply:
[Clicking on this](https://www.powershellgallery.com/packages/Az) goes to `0.2.2` for me.

Reply:
For me it goes to 0.1.0, I wonder if it's just cache.

Reply:
yep, just cache, I'll change it.